custom_components/adaptive_lighting/switch.py

```diff
@@ -786,6 +925,10 @@ async def _adapt_light(
         force: bool = False,
         context: Context | None = None,
     ) -> None:
+        brightness = None
```

Comment:
Are you sure setting these values is needed?

Reply:
Setting those was the easiest way to execute the following code later in that func:
```
        # used for alt_detect_method
        self._last_adapted_state = {}
        if brightness:
            self._last_adapted_state[ATTR_BRIGHTNESS] = brightness
        if color_temp_kelvin:
            self._last_adapted_state[ATTR_COLOR_TEMP_KELVIN] = color_temp_kelvin
        if rgb_color:
            self._last_adapted_state[ATTR_RGB_COLOR] = rgb_color
        self._manual_lights[light]['timer'] = 0
```
without declaring initially to None I would get:
NameError: name 'brightness' is not defined